interface/src/app/status/SystemLog.tsx

```diff
@@ -87,6 +87,7 @@ const SystemLog = () => {
 
   const [logEntries, setLogEntries] = useState<LogEntry[]>([]);
   const [lastIndex, setLastIndex] = useState<number>(0);
```

Comment:
You can remove the last_index check, it was to prevent double messages, but this is solved.